demo/roles/openfoam/tasks/main.yml

```diff
@@ -0,0 +1,40 @@
+- name: Install docker dependencies
+  become_user: root
```

Comment:
we need

```
become: yes
```

This is relying on become: yes being executed before this statement, which make not be the case depending on where this role is executed.